include/aspect/parameters.h

```diff
@@ -202,6 +202,109 @@ namespace aspect
      */
 
     /**
+     * @name Formulation settings
+     * @{
+     */
+    struct CombinedFormulation
```

Comment:
I think this might actually be easier to do with a `namespace`. This way you don't have to mark the function as `static` (though you then either need to put it into the .cc file -- my preference -- or make it `inline`).